__tests__/app.test.js

```diff
@@ -49,4 +49,40 @@ describe("nc-news", () => {
         });
     });
   });
+
+  describe("GET /api/articles/:article_id", () => {
+    test("GET 200: returns specific article when given valid id", () => {
```

Comment:
You're expecting a specific article with the id of 1...and yet expecting any returned data. You should be expecting the correct id, correct title and so on.